src/main/java/uk/gov/ons/ssdc/supporttool/model/dto/CaseSearchResult.java

```diff
@@ -0,0 +1,14 @@
+package uk.gov.ons.ssdc.supporttool.model.dto;
+
+import java.util.Map;
+import java.util.UUID;
+import lombok.Data;
+
+@Data
+public class CaseSearchResult {
+  public UUID id;
+  public String caseRef;
+  public Map<String, String> sample;
+  public UUID collectionExerciseId;
```

Comment:
what use is the collex ID?

Reply:
I was considering making the collex column in the results link to the collection exercise, since we haven't done that I'll remove it here 👍